student.rb

```diff
@@ -1,14 +1,15 @@
-# student.rb
-
 require_relative 'person'
 
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  attr_reader :classroom
+
+  def initialize(id, classroom, age = 0, name = 'Unknown', parent_permission: true)
+    super(id, name, age, parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
   end
 
   def play_hooky
-    '¯\\(ツ)/¯'
+    '¯\(ツ)/¯'
   end
```

Comment:
- [Optional] Dear @Ochiengsteven. When setting the classroom for a student we should also add it to the classroom's students. I notice you implement this feature inside the initializer method, consider a case where the student is created without class, and later on we wanna assign a student to a classroom. So, I would suggest creating a separate method for doing so, Something like...

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```